<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input Layer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonSet1">
        <property name="text">
         <string>Parameter Set 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="radioButtonSet2">
        <property name="text">
         <string>Parameter Set 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2" colspan="3">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Parameter 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxP1">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>C</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsColorButton" name="colorButtonP1"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Parameter 1</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Parameter 2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxP2">
            <item>
             <property name="text">
              <string>Value 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Value 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Value 3</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QgsMapLayerComboBox" name="comboBoxMapLayer"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Summary (press OK to refresh)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPlainTextEdit" name="textBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QPushButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>-17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
